units/systemd-coredump@.service.in

```diff
@@ -19,9 +19,19 @@ Before=shutdown.target
 ExecStart=-@rootlibexecdir@/systemd-coredump
 Nice=9
 OOMScoreAdjust=500
+RuntimeMaxSec=5min
```

Comment:
this patch doesn't actually change this setting, i just reorder the bits a bit...

If you don't like the value then prep a new PR!